schema/codelists/contractStatus.csv

```diff
@@ -1,7 +1,9 @@
-Code,Title,Description
-pending,Pending,"This contract has been proposed, but is not yet in force. It might be awaiting signature."
-active,Active,"This contract has been concluded by all the parties, and is now legally in force."
-cancelled,Cancelled,This contract has been cancelled prior to being concluded.
-terminated,Terminated,"This contract was concluded and in force, and has now come to a close. This might be due to the successful completion of the contract, or might be early termination due to some non-completion."
-terminatedEarly,Terminated early,"This contract was concluded and in force, and is terminated early due to some non-completion."
-terminatedSuccessfully,Terminated successfuly,"This contract was concluded and in force, and is terminated due to its successful completion."
+Code,Title,Description,Deprecated,Deprecation note
+pending,Pending,"This contract has been proposed, but is not yet in force. It might be awaiting signature.",1.2,
+active,Active,"The award has been made and the standstill period, if any, has ended. The contract has not yet terminated.",,
+terminated,Terminated,"The contract was concluded and in force, and has now come to a close. This might be due to the successful completion of the contract, or might be early termination due to some non-completion.",,
+terminatedSuccessfully,Terminated successfully,"The contract was concluded and in force, and is terminated due to its successful completion.",,
+terminatedEarly,Terminated early,"The contract was concluded and in force, and is terminated early due to some non-completion.",,
+cancelled,Cancelled,"The contracting process has been cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the award has been made and the standstill period, if any, has ended.",,
+unsuccessful,Unsuccessful,"The contract failed (e.g. the winning bidder(s) refused to sign the contract) after the award had been made and the standstill period, if any, had ended, but before the contract was signed and in force.",,
```

Comment:
Similar to above, we mention "in force" without any code causing this state. This is also the only place we refer to signature.

Reply:
Changed the one occurrence "signed and in force" to "concluded and in force" in https://github.com/open-contracting/standard/pull/1509/commits/ee1630e7554421a77272351a79538061b3b7321c. 

______

More broadly: I think we never properly discussed https://github.com/open-contracting/standard/issues/1160#issuecomment-887489821:

> I would have a tendency to drop 'pending' from both awardStatus and contractStatus. I'm assuming its main goal is to distinguish releases where an award has already been made or a contract already signed, but this information is clear from dateSigned and awardDate. The same argument could probably also be made against 'planned', 'active', 'complete' though, which would then lead to status changing into "reasons for failure"...?

The flow we're trying to model is award decision --> end of standstill (1) --> signature by last party --> contract in force --> contract completed/terminated (2).  In my proposal, at point 1, we set `awardStatus` = 'complete' and `contractStatus` to 'active'. Then we do nothing until 2, when we set `contractStatus` to 'terminated*'. 

Looking at it again, and if there is user need, we can keep/rephrase codes to cover:
- `awardStatus`: awarded, but still in standstill (by slightly rephrasing 1.1 'pending' from `awardStatus`) 
- `contractStatus`: signed, but not yet in force (by rephrasing 1.1 'pending' from `contractStatus` to correspond to this point and not to the next point)
- `contractStatus`: in force, but not yet terminated (by sticking closer to the 1.1 description of 'active') 

I guess the main point is: what do we want to track through `*status` and what is enough to have taken care of by another field (e.g. `dateSignature`, `dateAward`)? (Looking at it now, I'm actually more inclined to track it all in *status, especially since that is the status quo.)